src/domain/models/bertsum_transformer.py

```diff
@@ -10,30 +10,32 @@
 import numpy as np
 
 D_FFN = 2048
+D_MODEL_BERT = 768
 HEAD = 8
 DROPOUT = 0.1
 NUM_TLAYER = 2
+DEFAULT_BERT_MAX_LEN = 512
 
 
 class Summarizer(nn.Module):
     def __init__(self, device, config):
         super(Summarizer, self).__init__()
         self.device = device
-        if config.bert_cache:
-            bert_cache_dir = join(getcwd(), "bert_cache/bertmodel_save_pretrained")
-            self.bert = BertModel.from_pretrained(bert_cache_dir, local_files_only=True).to(self.device)
+        if not config.bert_cache:
+            self.bert = BertModel.from_pretrained("bert-base-uncased")
         else:
-            self.bert = BertModel.from_pretrained("bert-base-uncased").to(self.device)
+            self.bert = BertModel.from_pretrained(config.bert_cache, local_files_only=True)
+        self.bert.to(self.device)
```

Comment:
```suggestion
        self.bert = self.bert.to(self.device)
```
Le .to() est pas inplace, il faut storer le modèle retourner par to()